api/main_endpoints/util/registerUser.js

```diff
@@ -29,8 +29,7 @@ function testPasswordStrength(password) {
   return { success: mediumRegex.test(password), message: mediumMessage };
 }
 
-function getMemberExpirationDate(numberOfSemestersToSignUpFor = 0) {
-  /**
+/**
```

Comment:
add documentation for this too using the js documentation guide
you can also document parameters using @params ... 
all in the guide